config.py

```diff
@@ -75,6 +80,7 @@ class Test(Config):
 class Development(Config):
     DEBUG = True
     SESSION_COOKIE_SECURE = False
+    WTF_CSRF_ENABLED = False
```

Comment:
I'm not sure this is a good idea. It hides errors related to the `csrf_token`, which I believe the new forms are missing.